tests/telemetry/test_telemetry.py

```diff
@@ -60,3 +70,20 @@ def test_telemetry_enabledbydefault(duthost):
         if str(k) == "status":
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
+
+def test_telemetry_ouput(duthost, ptfhost):
+    """Run pyclient from ptfdocker and show gnmi server outputself.
+    """
+    logger.info('start telemetry output testing')
+    setup_telemetry_forpyclient(duthost)
+    # wait till telemetry is restarted
+    pytest_assert(wait_until(100, 10, duthost.is_service_fully_started, "telemetry"), "TELEMETRY not started")
+    logger.info('telemetry process restarted. Now run pyclient on ptfdocker')
+    dut_ip = duthost.setup()['ansible_facts']['ansible_eth0']['ipv4']['address']
```

Comment:
what about loopback reachability? Do we need to test and make sure the telemetry service can be reachable via loopback?

Reply:
Separate PR to check loopback reachability based on offline discussion